packages/node/src/lib/novu.ts

```diff
@@ -61,7 +61,14 @@ export class Novu extends EventEmitter {
         apiKey = args[0];
       }
     } else {
-      apiKey = getEnvVariable('NOVU_API_KEY');
+      apiKey =
+        getEnvVariable('NOVU_SECRET_KEY') || getEnvVariable('NOVU_API_KEY');
+    }
+
+    if (!apiKey) {
+      throw new Error(
+        'Missing secret key. Set the NOVU_SECRET_KEY environment variable or pass an apiKey to new Novu(apiKey) constructor.'
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ WE could also rename the `apiKey` parameter, otherwise it is confusing to use secret key and apiKey in one sentence

Reply:
Done!